provider-service/vai/cmd/main.go

```diff
@@ -26,57 +27,43 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+
 	ctx := logr.NewContext(context.Background(), logger)
 
-	config, err := configLoader.LoadConfig(ctx)
+	baseConfig, err := baseConfig.LoadConfig(ctx)
 	if err != nil {
 		panic(err)
 	}
+	logger.Info(fmt.Sprintf("loaded base config: %+v", baseConfig))
 
-	k8sClient, err := NewK8sClient()
+	vaiProviderConfig, err := config.LoadVAIProviderConfig()
 	if err != nil {
+		logger.Error(err, "failed to load provider config", "provider", baseConfig.ProviderName, "namespace", baseConfig.Pod.Namespace)
 		panic(err)
 	}
+	logger.Info(fmt.Sprintf("loaded VAI provider config: %+v", vaiProviderConfig))
 
-	vaiConfig := vaiConfig.VAIProviderConfig{
-		Name: config.ProviderName,
-	}
+	RunServer(ctx, vaiProviderConfig, baseConfig)
```

Comment:
In vai main.go we just do `server.Start()`, but in kfp main.go we run it as a go routine. Is there a reason for that or just oversight?